src/component/Calculator.jsx

```diff
@@ -2,41 +2,56 @@ import React from 'react';
 import './Calculator.css';
 import Container from '@mui/material/Container';
 import { Box } from '@mui/material';
+import calculate from '../logic/calculate';
 
 export default class Calculator extends React.Component {
   constructor() {
     super();
-    this.state = {};
+    this.clickHandler = this.clickHandler.bind(this);
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+  }
+
+  clickHandler(event) {
+    this.setState((state) => calculate(state, event.target.textContent));
   }
 
   render() {
+    const { total, next, operation } = this.state;
     return (
       <div>
         <Box m={15} />
         <Container maxWidth="xs">
           <div className="calculator-container">
             <div className="result-container">
-              <h1 className="result">0</h1>
+              <h1 className="result">
+                {total || ''}
+                {operation || ''}
+                {next || ''}
+              </h1>
             </div>
```

Comment:
- [ ] [OPTIONAL] Well done implementing the operations of the calculator, however, you need to fix the style for the screen `result-container` class. Please ensure it aligns properly.
#### Your Design
![Screenshot from 2022-03-17 23-43-42](https://user-images.githubusercontent.com/58692590/158906760-118ce846-99de-4a80-9fba-6de1d51b739a.png)